data/channels.csv

```diff
@@ -2084,9 +2088,9 @@ AutonomaTV.cl,Autonoma TV,,,,CL,CL-AR,Temuco,general,FALSE,,,,https://autonomatv
 AutonomaTV.uy,Autonoma TV,Autónoma TV,,,UY,,Paysandú,entertainment,FALSE,,,,https://www.autonoma.uy/,https://i.imgur.com/G5hB8bx.jpeg
 AutoPlus.ru,Auto Plus,Авто Плюс,,,RU,,Moscow,auto,FALSE,,,,https://www.autoplustv.ru/,https://i.imgur.com/sVc8cFt.png
 AvaEntertainment.iq,Ava Entertainment,طریف مصیف القدیم,,Mostafa Bayram,IQ,,Erbil,entertainment,FALSE,2019-03-20,,,http://www.avaentertainment.net/,https://i.imgur.com/me5igxY.jpg
-AVAFamily.iq,AVA Family,آوا فامیلی,,Mostafa Bayram,IQ,,Erbil,family,FALSE,2019-03-20,,,https://avafamily.avatv.live/,https://www.lyngsat.com/logo/tv/aa/ava-family-iq.png
+AVAFamily.ir,AVA Family,آوا فامیلی,,Mostafa Bayram,IQ,,Erbil,family,FALSE,2019-03-20,,,https://avafamily.avatv.live/,https://www.lyngsat.com/logo/tv/aa/ava-family-iq.png
 AvanceHD.hn,Avance HD,,,,HN,,Tegucigalpa,education,FALSE,,,,https://avancehd.tv/,https://i.imgur.com/gEiTlsC.png
-AvangTV.us,Avang TV,آونگ,,,US,US-CA,Los Angeles,music,FALSE,1992-01-01,,,http://www.avang.com/,https://i.imgur.com/3I1n7fO.png
+AvangTV.ir,Avang TV,آونگ,,,US,US-CA,Los Angeles,music,FALSE,1992-01-01,,,http://www.avang.com/,https://i.imgur.com/3I1n7fO.png

```

Comment:
According to its official YouTube page, the channel is broadcast from the United States.

https://www.youtube.com/user/avangmusic